api.planx.uk/send/bops.ts

```diff
@@ -12,6 +12,26 @@ interface SendToBOPSRequest {
   };
 }
 
+/**
+ * @swagger
+ * /bops/{localAuthority}:
+ *  post:
+ *    summary: Submits an application to the Back Office Planning System (BOPS)
+ *    description: Submits an application to the Back Office Planning System (BOPS)
+ *    tags:
+ *      - submissions
+ *    parameters:
+ *      - $ref: '#/components/parameters/localAuthority'
+ *    security:
+ *      - bearerAuth: []
+ *    requestBody:
+ *      description: This endpoint is only called via Hasura's scheduled event webhook, so body is wrapped in a `payload` key
+ *      required: true
+ *      content:
+ *        application/json:
+ *          schema:
+ *            $ref: '#/components/schemas/SessionPayload'
+ */
```

Comment:
was getting curious about documenting POSTs too - so just went for an MVP `submissions` section of docs too: 
![Screenshot from 2023-08-02 20-39-03](https://github.com/theopensystemslab/planx-new/assets/5132349/54737934-d88e-40bb-afad-7436b4510f66)

next steps i imagine here:
- correct auth (JWT doesn't quite feel accurate for these?)
- mock response statuses and data
- much more detailed human `description` - eg our submission endpoints are responsible for third party auth, generating payload, updating audit tables, etc. A lot of this lives in code comments, but could easily move into swagger docs here

Reply:
Yeah this is correct - JWT is not correct as these requests come via Hasura so it's more like an API key or header than validates this.